src/common/styles/GlobalStyles.tsx

```diff
@@ -46,6 +46,10 @@ const GlobalStyles = createGlobalStyle`
       padding: 0;
   }
 
+  body {
+    overscroll-behavior-x: none; /* Prevent the page redirection on swipe */
```

Comment:
It's not supported in safari yet. see [here](https://developer.mozilla.org/en-US/docs/Web/CSS/overscroll-behavior)